lib/OrdinaryDiffEqDifferentiation/src/alg_utils.jl

```diff
@@ -83,13 +83,12 @@ function DiffEqBase.prepare_alg(
 
         cs = ForwardDiff.pickchunksize(x)
         return remake(alg,
-            autodiff = AutoForwardDiff(
-                chunksize = cs))
+            autodiff = AutoForwardDiff{cs}(nothing))
```

Comment:
This will still be type unstable (and aparently we lost the functionbarrier that was supposed to stabilize this?)